keyvi/src/cpp/dictionary/fsa/internal/bit_vector_64.h

```diff
@@ -84,12 +84,12 @@ final {
 
       if (bitPosition == 0)
       {
-        for (auto i = 0; i<write_length; i++){
+        for (size_t i = 0; i<write_length; i++){
```

Comment:
In general it's a good practice to do pre-increment (++i) if we don't need the old value, as we avoid a temporary variable creation.